tests/integration/records/test_records_views.py

```diff
@@ -128,12 +131,13 @@ def test_literature_citations_api_with_results(isolated_api_client):
     assert response.status_code == 200
     assert expected_metadata == result['metadata']
 
-    record._delete(force=True)
-    record_ref_1._delete(force=True)
-    record_ref_2._delete(force=True)
+    _delete_record('lit', 111)
+    _delete_record('lit', 222)
+    _delete_record('lit', 333)
+    db.session.commit()
```

Comment:
no need to double commit https://github.com/inspirehep/inspire-next/blob/cc784e3565f0458977e3eb299de03262bab8f095/tests/integration/helpers/utils.py#L53 :)